### PR TITLE
Add "Sign-in with Apple"

### PR DESCRIPTION
**Pull request by riasvdv**
This works in pretty much the same way as the Github socialite integration works. Only this requires a custom one (thanks to https://github.com/GeneaLabs/laravel-sign-in-with-apple & https://github.com/m1guelpf/laravel-apple-login) for providing inspiration.

image

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -32,17 +32,35 @@ function showGitHubAuthWindow() {
...
-                <h3 class="title-sm mb-4">
-                    Via GitHub
-                </h3>
-                <a href="{{ route('github-login') }}" onclick="event.preventDefault(); showGitHubAuthWindow()">
-                    <x-button>
-                        <span class="mr-3 icon w-6 opacity">
+
+                <div class="grid grid-cols-1 sm:grid-cols-2 gap-12">
+                    <div>
+                        <h3 class="title-sm mb-4">
```
Comment (willemvb) [inline resources/views/auth/login.blade.php:38]: @riasvdv Maybe we can drop these headings?

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -32,17 +32,35 @@ function showGitHubAuthWindow() {
...
                         </span>
-                        Log in with GitHub
-                    </x-button>
-                </a>
+                                Log in with GitHub
+                            </x-button>
+                        </a>
+                    </div>
+                    <div>
+                        <h3 class="title-sm mb-4">
```
Comment (willemvb) [inline resources/views/auth/login.blade.php:51]: This one as well

**Review by willemvb — COMMENTED** [review]
Looking good!

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (riasvdv) [reply resources/views/auth/login.blade.php]: Yeah that's better!

**Pull request merged**